firefox-cliqz_privacy_notice/de.md

```diff
@@ -60,7 +60,7 @@ Firefox mit Cliqz stellt automatisch eine Verbindung mit Cliqz her, um Suchergeb
 * __Technische Daten für Updates__: Desktop-Versionen von Firefox prüfen in der Regel auf Browser-Updates, indem sie eine Verbindung mit Mozilla-Servern herstellen. Ihre Firefox-Version, Sprache und das Gerätebetriebssystem werden herangezogen, um die korrekten Updates anzuwenden. Mobilversionen von Firefox können eine Verbindung mit einem anderen Dienst herstellen, wenn Sie einen solchen zum Herunterladen und Installieren von Firefox verwendet haben [Weitere Informationen](https://support.mozilla.org/kb/how-stop-firefox-automatically-making-connections#w_auto-update-checking).
 {: #auto-updates }
 
-* __Technische Daten für Add-on-Blockierliste__: Firefox für Desktop und Android stellen von Zeit zu Zeit eine Verbindung mit Mozilla her, um Sie und Andere vor schädlichen Add-ons zu schützen.  Ihre Firefox-Version und Sprache, das Gerätebetriebssystem und die Liste installierter Add-ons werden benötigt, um die Add-on-Blockierliste anzuwenden und zu aktualisieren. [Weitere Informationen](https://blog.mozilla.org/addons/how-to-opt-out-of-add-on-metadata-updates/).
+* __Technische Daten für Add-on-Blockierliste__: Firefox für Desktop und Android stellen von Zeit zu Zeit eine Verbindung mit Mozilla her, um Sie und Andere vor schädlichen Add-ons zu schützen.  Ihre Firefox-Version und Sprache, das Gerätebetriebssystem und die Liste installierter Add-ons werden benötigt, um die Add-on-Blockierliste anzuwenden und zu aktualisieren. [Weitere Informationen](https://support.mozilla.org/kb/how-stop-firefox-making-automatic-connections).
```

Comment:
For German, it should be this: https://support.mozilla.org/de/kb/Firefox-baut-unaufgeforderte-Verbindungen-auf.

Reply:
The language neutral link will be redirected there for German:
https://support.mozilla.org/kb/how-stop-firefox-making-automatic-connections
-> https://support.mozilla.org/de/kb/how-stop-firefox-making-automatic-connections
-> https://support.mozilla.org/de/kb/Firefox-baut-unaufgeforderte-Verbindungen-auf 😺